dangerfile.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-// import { danger, warn } from "danger"
+import { danger, warn, markdown } from "danger"
```

Comment:
It is the case, yes, but its fakey - this is in there for tooling -> https://github.com/danger/danger-js/blob/master/source/runner/DangerfileRunner.js#L90

This means that flow/TS will give you the right autocompletions + documentation. The runner will just comment it out before evaling the DAngerfile